satip/goes_download_manager.py

```diff
@@ -0,0 +1,90 @@
+"""
+Script for downloading GOES data.
+"""
+
+import datetime
+import logging
+import os
+
+from goes2go import GOES
+
+
+class GOESDownloadManager:
```

Comment:
This looks good! As mentioned in #222, I think the best would be for this to be integrated into the main DownloadManager, and a parameter that specifies the provider used to switch between EUMETSAT and Goes imagery.